wallet/src/repository/wallet/mod.rs

```diff
@@ -1001,6 +1016,83 @@ where
         self.create_transaction_components(state, value, fee, recipient, false)
     }
 
+    fn create_vt_body(
+        &self,
+        state: &mut State,
+        value: u64,
+        fee: u64,
```

Comment:
Can we make it clear that this is not nanowits but nanowits/weight unit?

```suggestion
        weighted_fee: u64,
```

Reply:
as we have being speaking, both fee and weighted_fee exist now, the latter being an option

Reply:
So create_transaction_components always treats this as a fee and not a weighted fee. However, build_vt_body and build_dr_body implement the algorithms to make this a weighted fee